backend/config/settings.py

```diff
@@ -27,6 +27,8 @@
 
 ALLOWED_HOSTS = ['*']
 
+CORS_ORIGIN_ALLOW_ALL = True
```

Comment:
Had to add this to be able to call our api from the frontend container. Anyone know of a better restriction though?

Reply:
Might not need to restrict it since this is just a grad school project. We're actually already using that dependency.

Reply:
Yea, whatever you prefer. We can do it and if it becomes a pain we can open it back up.

Reply:
Sounds good, I added c4d3703 to restrict allowed origins to localhost. That's a good point, if it's a hassle we can allow all again in the future.